src/layers/embedding.h

```diff
@@ -12,8 +12,10 @@ class FactoredVocab;
 // EncoderDecoderLayerBase, which knows to pass on all required parameters from options.
 class Embedding : public LayerBase, public IEmbeddingLayer {
   Expr E_;
+  Expr FactorEmbMatrix_;
```

Comment:
Please add an inline comment what is it/what is it for.

Reply:
done